githubactions-db/files/etc/mysql/conf.d/custom.cnf

```diff
@@ -1,26 +1,31 @@
 
 [mysqld]
+# Tweak optimizer
+optimizer_switch='mrr=on'
+optimizer_switch='mrr_cost_based=off'
+
+# Mount datadir onto memory
+datadir=/dev/shm/mysql
 
-# Performance tweaks
-# see https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/mysql/images/*/Dockerfile
-# see https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/mysql/images/*/ram/Dockerfile
-collation-server               = utf8mb4_unicode_ci
-init-connect                   = "SET NAMES utf8mb4"
-character-set-server           = utf8mb4
```

Comment:
Default charset/collation depends on GLPI version. Thus `SET NAMES utf8mb4` is already done by GLPI itself.
So these values are counter-productive.